Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -314,6 +295,44 @@ class MessageGenerator {
     }
   }
 
+  func generateRuntimeSupport(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
```

Comment:
I presume the next PR would split this further into `generateProtoNameProviding` and `generateMessageImplementationBase` or some such?